CMakeLists.txt

```diff
@@ -47,16 +64,78 @@ add_executable(kcd2-386 EXCLUDE_FROM_ALL)
 target_sources(kcd2-386 PRIVATE "${PROJECT_SOURCE_DIR}/kauai/src/kcd2_386.c")
 target_include_directories(kcd2-386 PRIVATE $<TARGET_PROPERTY:kauai,INCLUDE_DIRECTORIES>)
 
-add_executable(chelp EXCLUDE_FROM_ALL)
-target_sources(chelp PRIVATE "${PROJECT_SOURCE_DIR}/kauai/tools/chelp.cpp")
+add_executable(chelp WIN32 EXCLUDE_FROM_ALL)
+target_sources(chelp PRIVATE
+    "${PROJECT_SOURCE_DIR}/kauai/TOOLS/CHTOP.CPP"
+    "${PROJECT_SOURCE_DIR}/kauai/TOOLS/CHELP.CPP"
+    "${PROJECT_SOURCE_DIR}/kauai/TOOLS/CHELPEXP.CPP"
+    "${PROJECT_SOURCE_DIR}/kauai/TOOLS/CHELP.RC"
+)
 target_include_directories(chelp PRIVATE $<TARGET_PROPERTY:kauai,INCLUDE_DIRECTORIES>)
 target_link_libraries(chelp PRIVATE $<TARGET_NAME_IF_EXISTS:kauai>)
 
+# Kauai test applications
+add_executable(ft WIN32 EXCLUDE_FROM_ALL)
+target_sources(ft PRIVATE
+    "${PROJECT_SOURCE_DIR}/kauai/SRC/FT.RC"
+    "${PROJECT_SOURCE_DIR}/kauai/SRC/FT.CPP"
+)
+target_link_libraries(ft PRIVATE kauai)
+
+add_executable(ut EXCLUDE_FROM_ALL)
+target_sources(ut PRIVATE
+    "${PROJECT_SOURCE_DIR}/kauai/SRC/UT.CPP"
+)
+target_link_libraries(ut PRIVATE kauai)
```

Comment:
Whats the difference between:
```
target_link_libraries(ut PRIVATE kauai)
```
and what @bruxisma used in:
```
target_link_libraries(chelp PRIVATE $<TARGET_NAME_IF_EXISTS:kauai>)
```

Reply:
That generator expression will link the Kauai library only if the Kauai target exists. I figured the Kauai target will always exist, so I didn't need the generator expression.

Reply:
I had used it at the time as `kauai` was not yet linking, but I was trying to get some stuff to just "build". Moving to this should be fine.